.sourcery-templates/DerivePartialIsos.stencil

```diff
@@ -9,8 +9,8 @@ import Prelude
 
       extension PartialIso where A == (
         {% for value in case.associatedValues %}
-          {% if value.type %}
-            {{ value.type.globalName }}{% if value.typeName.isOptional %}?{% endif %}
+          {% if value.type and value.type.parent %}
+            {{ value.type.name }}
```

Comment:
this is the only real change in this diff...

there are two places you can get a type name it seems, `value.type.name` and `value.typeName`. The latter omits generics and the former omits nested types. Let's just pray we never have nested generic types :/